php_pthreads.c

```diff
@@ -347,8 +347,8 @@ PHP_MINIT_FUNCTION(pthreads)
 #ifdef TCP_NODELAY
 	zend_declare_class_constant_long(pthreads_socket_entry, ZEND_STRL("TCP_NODELAY"), TCP_NODELAY);
 #endif
-	zend_declare_class_constant_long(pthreads_socket_entry, ZEND_STRL("PHP_NORMAL_READ"), PHP_NORMAL_READ);
-	zend_declare_class_constant_long(pthreads_socket_entry, ZEND_STRL("PHP_BINARY_READ"), PHP_BINARY_READ);
+	zend_declare_class_constant_long(pthreads_socket_entry, ZEND_STRL("PTHREADS_NORMAL_READ"), PTHREADS_NORMAL_READ);
+	zend_declare_class_constant_long(pthreads_socket_entry, ZEND_STRL("PTHREADS_BINARY_READ"), PTHREADS_BINARY_READ);
```

Comment:
imho the `PTHREADS` prefix isn't needed here since they are not global constants. They could be `Socket::NORMAL_READ` and `Socket::BINARY_READ`.

Reply:
@tpunt @dktapps what would you prefer? Just `Socket::NORMAL_READ` / `Socket::BINARY_READ` or prefixed with `PTHREADS_` ?

Reply:
I'm also in favour of `Socket::NORMAL_READ` and `Socket::BINARY_READ`.

Reply:
Renamed to `Socket::NORMAL_READ` and `Socket::BINARY_READ`